Src/AutoFixture/IFixtureExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Ploeh.AutoFixture
+{
+    public static class IFixtureExtensions
```

Comment:
I think we need some XML documentation there, otherwise the Code Analysis treats this as a warning (which then gets treated as an error during the build process).